packages/nodes-base/nodes/Google/Forms/GoogleFormsTrigger.node.ts

```diff
@@ -0,0 +1,192 @@
+import {
+	IDataObject,
+	ILoadOptionsFunctions,
+	INodeExecutionData,
+	INodePropertyOptions,
+	INodeType,
+	INodeTypeDescription,
+	IPollFunctions,
+	NodeApiError,
+	NodeOperationError,
+} from 'n8n-workflow';
+
+import { googleApiRequestAllItems } from './GenericFunctions';
+
+import moment from 'moment';
+
+export class GoogleFormsTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Google Forms Trigger',
+		name: 'googleFormsTrigger',
+		icon: 'file:googleForms.svg',
+		group: ['trigger'],
+		version: 1,
+		subtitle: '={{$parameter["formId"]}}',
```

Comment:
I would reference the event/trigger instead. The File ID is a long alphanumeric value, meaning it does not add useful information.

<img width="281" alt="image" src="https://user-images.githubusercontent.com/16496553/186768217-d0e420e2-40ea-4e2a-a2b2-db70f550c44c.png">

Reply:
I've changed it to `subtitle: '=Form ID: {{$parameter["formId"]}}',` to be consistent with the other form trigger(Typeform - also alphanumeric id). If we'd use `triggerOn` it would always be just "formSubmission" which IMO doesn't add useful information either